telegram/ext/_updater.py

```diff
@@ -207,60 +179,138 @@ def start_polling(
 
             allowed_updates (List[:obj:`str`], optional): Passed to
                 :meth:`telegram.Bot.get_updates`.
-            read_latency (:obj:`float` | :obj:`int`, optional): Grace time in seconds for receiving
+            read_timeout (:obj:`float` | :obj:`int`, optional): Grace time in seconds for receiving
                 the reply from server. Will be added to the ``timeout`` value and used as the read
                 timeout from server (Default: ``2``).
+            error_callback (Callable[[:exc:`telegram.error.TelegramError`], :obj:`None`], \
+                optional): Callback to handle :exc:`telegram.error.TelegramError` s that occur
+                while calling :meth:`telegram.Bot.get_updates` during polling. Defaults to
+                :obj:`None`, in which case errors will be logged.
 
         Returns:
-            :class:`queue.Queue`: The update queue that can be filled from the main thread.
+            :class:`asyncio.Queue`: The update queue that can be filled from the main thread.
+
+        Raises:
+            :exc:`RuntimeError`: If the updater is already running.
 
         """
-        with self.__lock:
-            if not self.running:
-                self.running = True
-
-                # Create & start threads
-                dispatcher_ready = Event()
-                polling_ready = Event()
-
-                if self.dispatcher:
-                    self._init_thread(self.dispatcher.start, "dispatcher", ready=dispatcher_ready)
-                self._init_thread(
-                    self._start_polling,
-                    "updater",
-                    poll_interval,
-                    timeout,
-                    read_latency,
-                    bootstrap_retries,
-                    drop_pending_updates,
-                    allowed_updates,
+        async with self.__lock:
+            if self.running:
+                raise RuntimeError('This Updater is already running!')
+            if not self._initialized:
+                raise RuntimeError('This Updater is not initialized!')
```

Comment:
maybe also give a hint on how to initialize the Updater? Same thing in Application too